sass/components/_on-this-page.scss

```diff
@@ -0,0 +1,36 @@
+/// internal page navigation
+/// i.e., the "On This Page" section
+
+
+.tree-menu .on-this-page {
+    grid-row: 2;
```

Comment:
I would create a `tree-menu__toc` (in `_tree-menu.scss`) and move this `grid-row` rule there. I think it composes better that way; that way `.on-this-page` could be used in another context.

So the idea is to have two classes:

```
<nav aria-label="On This Page" class="on-this-page tree-menu__toc">
```

- `on-this-page` will control the internal styling
- `tree-menu__toc` will control the layout in relation to `tree-menu`

Reply:
done :)